packages/dd-trace/test/setup/mocha.js

```diff
@@ -70,6 +70,7 @@ function withVersions (plugin, modules, range, cb) {
       .filter(instrumentation => instrumentation.name === moduleName)
       .forEach(instrumentation => {
         instrumentation.versions
+          .filter(version => !process.env.RANGE || semver.subset(version, process.env.RANGE))
```

Comment:
nitpick but maybe `TEST_RANGE` is a better name?

Reply:
I'm not sure it really helps provide more meaning to the variable. I also thought about `WITH_RANGE` `VERSION_RANGE` maybe. What it actually does is not even really a range, it provides lower and upper bounds _around_ a range. Really not sure what the best name for this would be.

Reply:
Let's keep `RANGE` for now as this can always be changed later, and the whole range filtering probably needs to be redone anyway.